scripts/BuildAndTest.ps1

```diff
@@ -117,12 +117,6 @@ function New-SigningDirectory {
             }
         }
     }
-
-    # Copy the viewer. Its name doesn't fit the pattern binary name == project name,
-    # so we copy it by hand.
-    foreach ($framework in $Frameworks.NetFX) {
-        Copy-Item -Force -Path $BinRoot\${Platform}_$Configuration\Sarif.Viewer.VisualStudio\Microsoft.Sarif.Viewer.dll -Destination $SigningDirectory\$framework
-    }
```

Comment:
>} [](start = 4, length = 1)

Yeah!!! That viewer was _so_ out of step with everything else, so it's great to have it in a separate reason even if there were no other reason than that.